### DISCUSSION TITLE
Support Aliases like click-aliases package from click-contrib repo

### DISCUSSION THREAD

**Discussion by chmuche** [fastapi/typer#710] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import typer

# Direct alias on the Typer instance
db_app = type.Typer(name="database", aliases=["db", "data", "sql"])

# Support on the `command` decorator
# main command name is `create` or the value of the `name` attribute
@db_app.command(aliases=["new", "add"]):
def create():
    """Create new database"""


main_app = type.Typer()

# Add a new Alias when register
main_app.add_typer(db_app, aliases=db_app.aliases + ["pg"])
```

### Description

Inspired by the click-contrib [click-aliases](https://github.com/click-contrib/click-aliases)

I think supporting aliases directly without the click complexity natively inside typer will be awesome.

I can write a PR to support this if you like.

### Operating System

Linux

### Typer Version

0.9.0

### Python Version

3.10

**Comment by steinerkelvin** [comment]
Would it be possible to instruct click to do this as a workaround in the present moment?

**Comment by sheldonldev** [comment]
I'll be happy to have this solution.

**Comment by davidonlaptop** [comment]
See this discussion here: https://github.com/fastapi/typer/issues/132
